libs/localization/src/lib/en.json

```diff
@@ -1316,6 +1316,25 @@
     },
     "Navigation": {
       "modelStatistics": "Model statistics"
+    },
+    "ModelOverview": {
+      "metricsDropdown": "Metric(s)",
+      "accuracy": "Accuracy",
+      "f1Score": "F1 score",
```

Comment:
I already see some of these metrics here:
https://github.com/microsoft/responsible-ai-toolbox/blob/9a2380386828986306f367db3b99408abcdbeede/libs/localization/src/lib/en.json#L256
I wonder is there some central section we can put them under, to reduce duplication of metric names (and any possible inconsistencies)?

Reply:
I was considering that initially. However, it seems like we don't have a full set somewhere and the strings don't exactly match either. Adding support for the metrics I need everywhere else is work I haven't budgeted for, so I am creating a special set tailored to what I need to create. As we move away from UI-calculated metrics towards SDK calculated metrics this will have to be overhauled anyway. Wdyt?

Reply:
I'll open an item to consolidate these. I think it needs to be addressed, too, but it may be better suited for the big data discussion.